examples/direct-llm-calls-with-multimodal-input/direct_llm_calls_with_multimodal_input.bal

```diff
@@ -0,0 +1,50 @@
+import ballerina/ai;
+import ballerina/io;
+import ballerinax/ai.anthropic;
+
+// Use `configurable` variables for the model name and API key.
+configurable anthropic:ANTHROPIC_MODEL_NAMES modelType = ?;
+configurable string apiKey = ?;
+
+// Create a model provider to interact with the Anthropic API.
+// The `temperature` parameter is optional, and defaults to 0.7 if not specified.
+final ai:ModelProvider model =
+    check new anthropic:ModelProvider(apiKey, modelType, temperature = 0);
```

Comment:
I think this is not a usecase where temparature=0

Reply:
IMO, that depends on the user's expectation and 0 can make sense too. Like I don't need it to be creative or random here. But I kinda agree with your offline comment about setting an extreme in the example, so updated it to 0.3 and explained temperature in the comment.